atlp-rwanda/vikings-ec-fe
# #184759903 Implement seller two factor auth

### What does this PR do?
This PR implements seller two factor auth

### Description of Tasks to be completed?
A seller must send authCode to login
### How should this be manually tested?
- Clone [repo](https://github.com/atlp-rwanda/vikings-ec-fe.git)
- Change into the directory **vikings-ec-fe**
- Checkout to the branch: `feature/seller-auth-184759903`
- Run the command `npm install` 
- Run the command `npm run start`
- Try login as a seller
- Check your email for authcode

### What are the relevant pivotal tracker stories?
[#184759903](https://www.pivotaltracker.com/story/show/184759903)

### Screenshots
<img width="1436" alt="Screenshot 2023-04-11 at 14 54 06" src="https://user-images.githubusercontent.com/83646585/231170350-a6f601b4-6294-4d38-ac1a-10155b145276.png">
